views/js/pciCreator/dev/audioRecordingInteraction/pciCreator.json

```diff
@@ -29,9 +29,7 @@
             "./runtime/img/controls.svg",
             "./runtime/img/mic.svg",
             "./runtime/js/workers/WavAudioEncoder.js",
-            "./runtime/js/workers/WavAudioEncoder.min.js",
-            "./runtime/js/workers/WebAudioRecorderWav.js",
-            "./runtime/js/workers/WebAudioRecorderWav.min.js"
+            "./runtime/js/workers/WebAudioRecorderWav.js"
```

Comment:
You've removed the file `runtime/js/workers/WebAudioRecorderWav.min.js` but not `runtime/js/workers/WavAudioEncoder.min.js`

Reply:
Currently there is no bundling script which responsible for these files. Since these files weren't updated recently, I removed them. There is already a `TODO` about this step: https://github.com/oat-sa/extension-tao-itemqti-pci/blob/master/views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/providers/webAudio.js#L37

Reply:
yes but you've actually removed (using git) only one of the 2 files